java-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/JavaParserFacade.java

```diff
@@ -270,6 +271,10 @@ private void solveArguments(Node node, NodeList<Expression> args, boolean solveL
         }
     }
 
+    public SymbolReference<ResolvedFieldDeclaration> solve(FieldAccessExpr fieldAccessExpr) {
```

Comment:
We should consider exposing this using the new API (see JavaSymbolSolver class). This would be an additional improvement, not something we necessarily need to do in this PR